src/coreclr/inc/switches.h

```diff
@@ -165,14 +165,11 @@
 #define CHAIN_LOOKUP
 #endif // FEATURE_VIRTUAL_STUB_DISPATCH
 
-#if !defined(FEATURE_PORTABLE_ENTRYPOINTS) && !defined(TARGET_X86)
+// FEATURE_PORTABLE_SHUFFLE_THUNKS depends on CPUSTUBLINKER that is de-facto JIT
+#if defined(FEATURE_JIT) && !defined(TARGET_X86)
 #define FEATURE_PORTABLE_SHUFFLE_THUNKS
```

Comment:
I have renamed this back since we have shuffle thunks that are not portable shuffle thunks, and the rename made it confusing